modules/openshift-developer-cli-attributes.adoc

```diff
@@ -1,3 +1,3 @@
 :odo-title-full: OpenShift Do
 :odo-title: odo
-:odo-ver: 1.0
\ No newline at end of file
+:odo-ver: 1.1.0
```

Comment:
wait what

Reply:
We're releasing 1.1.0 along with 4.3, correct? Want to make sure before this merged. Otherwise LGTM.